### DISCUSSION TITLE
Expose Span Name in Generations table UI

### DISCUSSION THREAD

**Discussion by JeongJuhyeon** [langfuse/langfuse#4856] [Ideas]
### Describe the feature or potential improvement

<img width="1284" alt="image" src="https://github.com/user-attachments/assets/2a92d6a9-305e-426b-b862-0a3f67f0e670" />

Reason 1: Adding generation names is not supported in all environments yet. For example, when using the Vercel SDK integration, there is no way to set one. Currently, [it works through the span name](https://langfuse.com/docs/integrations/vercel-ai-sdk#customization). But since that is not exposed in the Generations table UI,  it's very hard to search for multiple instances of the same prompt being run, e.g. for comparative analysis.

Reason 2: Without it there's not much reason to use span names as they only get exposed in one UI (the timeline) and you can't filter on them or do anything with them.

You can even make the column hidden by default like a lot of them already are.

**Comment by maxdeichmann** [comment]
Hi @JeongJuhyeon thanks a lot for the feedback! We actually shipped this today. If you have more feedback for us, let us know.

**Comment by JeongJuhyeon** [comment]
@maxdeichmann 
Thanks Max, super helpful!
